### PR TITLE
Add Elixir 1.11 and OTP 23

### PR DESCRIPTION
**Pull request by michaelstalker**
[Card](https://rentpath.atlassian.net/browse/SRV-2500)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 1. Download the source code from https://github.com/elixir-lang/elixir/releases/tag/v1.9.4 ,
```
Comment (pasha) [inline README.md:5]: Can you please update the link to latest release https://github.com/elixir-lang/elixir/releases/tag/v1.11.2

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
...
    substituting the desired version in the URL.
-2. Compute the SHA256 digest for the `.tar.gz` file with `sha256sum`.
+2. Compute the SHA256 digest for the `.tar.gz` file with `sha256sum`. You can also use `shasum -a 256 <filename>` on a Mac.
 3. Update `ELIXIR_VERSION` and `ELIXIR_VERSION_SHA` in `Dockerfile`.
 4. Update the base Erlang version in the `FROM` statement if appropriate.
 5. Run `docker build -t rentpath/rp_elixir:v1.9.4-alpine-otp-22 -t rentpath/rp_elixir:latest .`, substituting the desired version in the tag.
```
Comment (pasha) [inline README.md:10]: Can you please update docker image to the latest version.

**Review by pasha — APPROVED** [review]

**Review by Stratus3D — APPROVED** [review]
💥 :tada:

**Comment by Stratus3D** [comment]
Nit pick: It's still Elixir 1, just 1.11 now :)

**Pull request merged**